product_docs/docs/pem/9/upgrading/upgrading_pem_installation/upgrading_pem_installation_linux_rpm.mdx

```diff
@@ -8,105 +8,89 @@ redirects:
 - /pem/latest/pem_upgrade/upgrading_pem_installation/upgrading_pem_installation_linux_rpm/
 ---
 
-To upgrade PEM component software on Linux hosts, install a newer version of the PEM component native packages in the following order:
+To upgrade PEM component software on Linux hosts, install a newer version of the PEM component packages in the following order:
 
-1.  Invoke the PEM agent native package installer on each monitored node except the PEM server host.
-2.  Invoke the PEM server native package installer. it upgrades both the PEM server and the PEM agent that resides on the PEM server host.
+1.  Invoke the PEM agent package installer on each monitored node except the PEM server host.
+2.  Invoke the PEM server package installer. it upgrades both the PEM server and the PEM agent that resides on the PEM server host.
 
 During an installation, the component installation automatically detects an existing installation and performs an upgrade. After upgrading the PEM agent and server, you can upgrade SQL Profiler if required. That step is platform specific.
 
 !!! Note
     If you already configured or are planning to configure any shell/batch script run by a Linux agent that's upgraded from any earlier version to version 7.11 or later, you must speciy the user for the `batch_script_user` parameter in the `agent.cfg` file. We strongly recommended that you use a non-root user to run the scripts. Using the root user can result in compromising the data security and operating system security. However, if you want to restore the pemagent to its original settings using a root user to run the scripts, then you must set the `batch_script_user` parameter to `root`.
 
-## Prerequisites to upgrade a PEM installation on Linux host
+## Upgrading a PEM server installation
 
-PEM depends on third-party components from the vendor repository, including python3, libboost, openssl, snmp++, and libcurl. To ensure these components are up to date, update your operating system using platform-specific commands. 
+If you initially used packages to install your PEM server, you can use packages to upgrade your PEM server. The commands to upgrade are platform specific. 
```

Comment:
This line is not required. We can start this description with, "The commands to upgrade PEM Server are platform specific."